EIPS/EIP-X-add-token-uri-kylanhurt.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
+
+    EIP: <to be assigned>
```

Comment:
Please rename this to `eip-961.md` and set the number as 961 here.

Reply:
Okay, will do as soon as I can get around to it.